docs/intro/tutorial.rst

```diff
@@ -55,318 +55,333 @@ This will create a ``tutorial`` directory with the following contents::
 
-Extracting Items
-----------------
+A shortcut to the start_requests method
+---------------------------------------
+Instead of implementing a :meth:`~scrapy.spiders.Spider.start_requests` method
+that generates :class:`scrapy.Request <scrapy.http.Request>` objects from URLs,
+you can just define a :attr:`~scrapy.spiders.Spider.start_urls` class attribute
+with a list of URLs. This list will then be used by the default implementation
+of :meth:`~scrapy.spiders.Spider.start_requests` to create the initial requests
+for your spider::
 
-Introduction to Selectors
-^^^^^^^^^^^^^^^^^^^^^^^^^
+    import scrapy
 
-There are several ways to extract data from web pages. Scrapy uses a mechanism
-based on `XPath`_ or `CSS`_ expressions called :ref:`Scrapy Selectors
-<topics-selectors>`.  For more information about selectors and other extraction
-mechanisms see the :ref:`Selectors documentation <topics-selectors>`.
 
-.. _XPath: https://www.w3.org/TR/xpath
-.. _CSS: https://www.w3.org/TR/selectors
+    class QuotesSpider(scrapy.Spider):
+        name = "quotes"
+        start_urls = [
+          'http://quotes.toscrape.com/page/1/',
+          'http://quotes.toscrape.com/page/2/',
+        ]
 
-Here are some examples of XPath expressions and their meanings:
+        def parse(self, response):
+            page = response.url.split("/")[-2]
+            filename = 'quotes-%s.html' % page
+            with open(filename, 'wb') as f:
+                f.write(response.body)
 
-* ``/html/head/title``: selects the ``<title>`` element, inside the ``<head>``
-  element of an HTML document. Equivalent CSS selector: ``html > head > title``.
+The :meth:`~scrapy.spiders.Spider.parse` method will be called to handle each
+of the requests for those URLs, even though we haven't explicitly told Scrapy
+to do so. This happens because :meth:`~scrapy.spiders.Spider.parse` is Scrapy's
+default callback method, which is called for requests without an explicitly
+assigned callback.
 
-* ``/html/head/title/text()``: selects the text inside the aforementioned
-  ``<title>`` element. Equivalent CSS selector: ``html > head > title ::text``.
 
-* ``//td``: selects all the ``<td>`` elements from the whole document.
-  Equivalent CSS selector: ``td``.
+Extracting data
+---------------
 
-* ``//div[@class="mine"]``: selects all ``div`` elements which contain an
-  attribute ``class="mine"``. Equivalent CSS selector: ``div.mine``.
+The best way to learn how to extract data with Scrapy is trying selectors
+using the shell :ref:`Scrapy shell <topics-shell>`. Run::
 
-These are just a couple of simple examples of what you can do with XPath, but
-XPath expressions are indeed much more powerful. To learn more about XPath, we
-recommend `this tutorial to learn XPath through examples
-<http://zvon.org/comp/r/tut-XPath_1.html>`_, and `this tutorial to learn "how
-to think in XPath" <http://plasmasturm.org/log/xpath101/>`_.
+    scrapy shell http://quotes.toscrape.com/page/1/
```

Comment:
let's write it as `scrapy shell 'http://quotes.toscrape.com/page/1/'`; "&" handling in console is an endless source of confusion.